src/guide/forms.md

```diff
@@ -1,5 +1,7 @@
 # Form Input Bindings
 
+<VideoLesson href="https://vueschool.io/lessons/user-inputs-vue-devtools-in-vue-3?friend=vuejs" title="Learn how to handle form inputs on Vue School">Learn how to work with form inputs with a free Vue School lesson</VideoLesson>
```

Comment:
For this video, as a new student I would expect a much more in-depth covering of form inputs across different input items, along with a more in-depth explanation of what v-model is. This video shows that v-model enables two-way binding, but then spends the rest of the video talking about DevTools. This seems mismatched for the docs.